### DISCUSSION TITLE
how to import FabricObject on v6?

### DISCUSSION THREAD

**Discussion by wuxingping** [fabricjs/fabric.js#8648] [General]
i am glad to know that typescript be supported in fabric v6. but i donot know how to import any type.for example. how to import FabricObject?

**Comment by ShaMan123** [comment]
master can be considered unofficially in beta
```
import * as fabric from 'fabric'
import { FabricObject } from 'fabric'

fabric.Object === FabricObject
```

**Comment by ShaMan123** [comment]
beta is published
